examples/variational_autoencoder.py

```diff
@@ -27,7 +27,7 @@
 
 def sampling(args):
     z_mean, z_log_var = args
-    epsilon = K.random_normal(shape=(batch_size, latent_dim), mean=0.,
+    epsilon = K.random_normal(shape=(K.shape(z_mean)[0], K.shape(z_mean)[1]), mean=0.,
```

Comment:
How about:

    epsilon = K.random_normal(shape=K.shape(z_mean), mean=0.,

this works on Tensorflow CPU.